### PR TITLE
fix(package.json): fixed storybook npm scripts

### PR DESCRIPTION
**Pull request by rannyeli**
## What I did

- [ ] BREAKING CHANGE
- [ ] feature request
- [x] bug fix
- [x] documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,3 +72,12 @@ or
 ```json
   npx lerna exec -- npm publish
 ```
+## Storybook
+1. Run all packages storybooks
+```json
```
Comment (syncush) [inline README.md]: why json, use sh

File: README.md [hunk 2]
```diff
@@ -72,3 +72,12 @@ or
...
+  yarn run storybook:all
+``` 
+1. Run single composed storybook which containing all storybooks
+```json
```
Comment (syncush) [inline README.md]: same, use sh

File: README.md [hunk 3]
```diff
@@ -72,3 +72,12 @@ or
 ```json
   npx lerna exec -- npm publish
 ```
+## Storybook
+1. Run all packages storybooks
+```json
+  yarn run storybook:all
+``` 
+1. Run single composed storybook which containing all storybooks
```
Comment (syncush) [inline README.md]: it needs to be 2, no?

File: README.md [hunk 4]
```diff
@@ -72,3 +72,12 @@ or
...
+```json
+  yarn run storybook-composition
+```
```
Comment (syncush) [inline README.md]: It is best if we add a comment that stage 1 will open 2 storybooks tabs and the terminal will be captured to the servers.

and that we need to run stage 2 in a different terminal for a composed storybook while the other 2 are still running.

**Review by syncush — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rannyeli) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (rannyeli) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (rannyeli) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (rannyeli) [reply README.md]: done

**Review by syncush — APPROVED** [review]

**Pull request merged**